### PR TITLE
Update dependencies to latest version

### PR DESCRIPTION
**Pull request by maksimkurb**
- [x] Updated dependencies, especially `@sentry/minimal` and `@sentry/hub` (was updated from `4.6.4` to `5.2.0`)
- [x] Fix tests

### CODE REVIEW

File: test/hello.e2e.spec.ts [hunk 1]
```diff
@@ -1,31 +1,30 @@
-import * as request from 'supertest';
```
Comment (mentos1386) [inline test/hello.e2e.spec.ts]: Can you revert any code style changes and leave only actual code changes.

Comment (maksimkurb) [reply test/hello.e2e.spec.ts]: Ah, sorry, editor did it for me 😅

**Comment by maksimkurb** [comment]
> Can you revert any code style changes and leave only actual code changes.

Fixed with 000bf05da9f83b1df22b333566a1727ed5992246

_P.S. I am correct, that [I can't re-request code review on GitHub](https://stackoverflow.com/questions/40893008/how-to-resume-review-process-after-updating-pull-request-at-github)? That's counter-intuitive_

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nest-raven",
-  "version": "4.0.0",
+  "version": "4.1.0",
```
Comment (mentos1386) [inline package.json]: More of a question than an issue.

Do you think if we update mayor version of `@sentry/minimal`. Should we update mayor version of `nest-raven` as well?

Or is it fine to leave it like this and npm/yarn takes care of dependencies (warns users that they need 5.x sentry or something)?

Comment (maksimkurb) [reply package.json]: Yes, probably we should bump major version due to breaking changes (I personally don't read output of npm install carefully each time) and use `npm deprecate` to inform user about ability to switch to Sentry@5, thanks for notice.

Should I bump version to 5.0.0 and cleanup commit history?

Comment (mentos1386) [reply package.json]: Nah it's fine, I'll merge and change it later. Thanks for PR!

**Pull request merged**